Standardize File Size Units to MB

yenaing-oo opened the pull request:
Closes https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/issues/305

danloveg reviewed (changes requested):
Everything looks good, except I noticed that the source types, source roles, and rights types were missing from the About page. Not sure when that regression happened, but can you make it so the tests that I added in 7e4df43173907bde0e19d97f61d33d3dadf32bed pass?

I figured rather than explain what was wrong, I'd add some tests that explain the problem. That way, if this happens again in the future the tests will catch this issue!

yenaing-oo commented:
> Everything looks good, except I noticed that the source types, source roles, and rights types were missing from the About page. Not sure when that regression happened, but can you make it so the tests that I added in [7e4df43](https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/commit/7e4df43173907bde0e19d97f61d33d3dadf32bed) pass?
> 
> I figured rather than explain what was wrong, I'd add some tests that explain the problem. That way, if this happens again in the future the tests will catch this issue!

Thanks for making a test class for the About page. I unintentionally left out that missing context in the changes dealing with splitting `views.py` into separate modules: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/commit/53a3a5c55c3e0aff4c55902887fe510e8a9fb810#diff-8e91c264512719a373ce645bd929218044d401674dfcecb4153f0b07fa487834R24

I've added it back here (https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/378/commits/c5c01595ebb5413c3a9bc209d4f03c22b6c2e30a) and the tests you added are now passing.

danloveg commented:
> > Everything looks good, except I noticed that the source types, source roles, and rights types were missing from the About page. Not sure when that regression happened, but can you make it so the tests that I added in [7e4df43](https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/commit/7e4df43173907bde0e19d97f61d33d3dadf32bed) pass?
> > I figured rather than explain what was wrong, I'd add some tests that explain the problem. That way, if this happens again in the future the tests will catch this issue!
> 
> Thanks for making a test class for the About page. I unintentionally left out that missing context in the changes dealing with splitting `views.py` into separate modules: [53a3a5c#diff-8e91c264512719a373ce645bd929218044d401674dfcecb4153f0b07fa487834R24](https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/commit/53a3a5c55c3e0aff4c55902887fe510e8a9fb810#diff-8e91c264512719a373ce645bd929218044d401674dfcecb4153f0b07fa487834R24)
> 
> I've added it back here ([c5c0159](https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/commit/c5c01595ebb5413c3a9bc209d4f03c22b6c2e30a)) and the tests you added are now passing.

Great, thanks for fixing that up. Everything else looks good
